Formula/qca.rb

```diff
@@ -4,8 +4,8 @@ class Qca < Formula
   head "https://invent.kde.org/libraries/qca.git"
 
   stable do
-    url "https://github.com/KDE/qca/archive/v2.3.0.tar.gz"
-    sha256 "39aa18f0985d82949f4dccce04af3eb8d4b6b64e0c71785786738d38d8183b0a"
+    url "https://download.kde.org/stable/qca/2.3.1/qca-2.3.1.tar.xz"
+    sha256 "c13851109abefc4623370989fae3a745bf6b1acb3c2a13a8958539823e974e4b"
 
     # use major version for framework, instead of full version
```

Comment:
Has this been reported in the proper upstream repo?

Reply:
`download.kde.org` is recommended download source for all KDE software, it's actually a load balancer powered by MirrorBrain. And it was in this release announce: https://mail.kde.org/pipermail/kde-announce-apps/2020-July/005594.html

Reply:
Oh, sorry, wrong line. I had sent this commit [upstream](https://invent.kde.org/libraries/qca/-/merge_requests/34).

Reply:
Is there a way to use that as the source?

Reply:
Its actually was already merged.